### ISSUE TITLE
Color contrast inaccessibles for some states of buttons

### ISSUE THREAD

**Issue by Marine-Fabien** [adeo/mozaic-design-system#1162]
# 🐞 Styleguide bug report

### Is this a regression?

No

### Description

Some contrasts are inaccessibles for buttons :
> mc-button--bordered > state active 
> mc-button--bordered > state hover
> mc-button--bordered-danger > state active
> all mc-button > state disabled

## 🔬 Steps to reproduce the issue

Check the colour contrast with any tool designed for this purpose

## 📷 Screenshot (optional - it might be a gif)
<img width="1215" alt="Capture d’écran 2022-06-30 à 17 07 11" src="https://user-images.githubusercontent.com/77010633/176714012-521fa11b-2742-48c8-b3fa-c46cd31224c1.png">

1. **mc-button--bordered > state active**

<img width="334" alt="Capture d’écran 2022-06-30 à 17 09 52" src="https://user-images.githubusercontent.com/77010633/176714144-1e4f76c1-7b87-4061-9039-c8620e5df23a.png">

2. **mc-button--bordered > state hover**

<img width="334" alt="Capture d’écran 2022-06-30 à 17 18 36" src="https://user-images.githubusercontent.com/77010633/176714572-32c4a608-fa68-4795-8d00-17e79c7c9daa.png">

3. **mc-button--bordered-danger > state active**

<img width="334" alt="Capture d’écran 2022-06-30 à 17 20 18" src="https://user-images.githubusercontent.com/77010633/176714845-a0d5d915-8172-458b-b37b-a5e2708af87b.png">

4. **all mc-button > state disabled**

<img width="334" alt="Capture d’écran 2022-06-30 à 17 21 00" src="https://user-images.githubusercontent.com/77010633/176714998-db6c383d-489c-4385-93d2-3f24dd92a3d2.png">

**Comment by caqueste** [comment]
Hello @Marine-Fabien 👋,
Thanks for the feedback. What are your recommendations for the Hover and Active states?
However, I don't understand your feedback on the disabled state. It seems to me that a disabled element doesn't need to be accessible. Can you tell me more about this point, please?

**Comment by Marine-Fabien** [comment]
Hello @caqueste,
I think you can try going up a colour tone on the mozaic palette for the target labels, the ratio should be good.
For clarification, regarding the "Disabled" button, accessibility is not only for the blind, if you are colour blind and have several difficulties to see all the contrasts, by respecting this minimum ratio, you allow the person to read the text on the button, as you read it.

**Comment by Marine-Fabien** [comment]
Yes, indeed @caqueste there is no legal obligation for disabled buttons, it is a choice between legality and human beings. It all depends on whether Adeo design system deals with accessibility to comply with the law or to help users.
